### PR TITLE
Added section for Professional Services 

### PR DESCRIPTION
**Pull request by eugeniop**
This PR adds a `/services` section describing Professional Services packages. 

@germanmatosas please review.

### CODE REVIEW

File: articles/services/performance-scalability.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+description: Project to assist customers with common performance testing needs, including a correct planning, and test automation guidance.
+---
+
+# Geo High Availability
```
Comment (mpaktiti) [inline articles/services/performance-scalability.md]: @eugeniop There are 2 "Geo High Availability" items, looks like this one is wrong.

https://auth0-docs-content-pr-2798.herokuapp.com/docs/services > Assess & Plan

image

File: articles/services/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  'services/scenario-guidance',
+  'services/code-review'
+] }) %>
+
+## Maintain & Improve
+
+<%= include('../_includes/_topic-links', { links: [
+  'services/pair-programming',
+  'services/private-saas-configuration',
+  'services/code-review',
```
Comment (mpaktiti) [inline articles/services/index.md:31]: @eugeniop code-review is listed twice, under **Assess & Plan** and **Maintain & Improve**. Just double-checking that this was not done by mistake.

File: articles/services/performance-scalability.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Estimated effort
+
+80 hours. 
+
+> Auth0 Professional Services are delivered in 40 hs. "blocks".
+
+## Typical work-streams
+
+* **Planning:** gather customer requirements for setting benchmarks.* **Performance Lab Design:** lab environment for testing, including network topology, artifacts required, load generation tools, automation scripts.
```
Comment (mpaktiti) [inline articles/services/performance-scalability.md]: @eugeniop You need to add an enter before the **Performance Lab Design** bullet. It displays like that:
- **Planning:** gather customer requirements for setting benchmarks.\* **Performance Lab Design:** lab environment for testing, including network topology, artifacts required, load generation tools, automation scripts.

File: articles/services/index.md (hunk 2 above)
Comment (eugeniop) [reply articles/services/index.md:31]: yes, that is correct. Same "package" in both,

**Comment by eugeniop** [comment]
Thanks Nate / Maria

File: articles/services/performance-scalability.md (hunk 1 above)
Comment (germanmatosas) [reply articles/services/performance-scalability.md]: Correct, this one should be "Performance and Scalability".

Comment (eugeniop) [inline articles/services/performance-scalability.md]: Dup title

Comment (eugeniop) [reply articles/services/performance-scalability.md]: Should be "Performance and Scalability"

**Pull request merged**